senders/webhook/webhook.go

```diff
@@ -7,9 +7,20 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/mitchellh/mapstructure"
 	"github.com/moira-alert/moira"
 )
 
+// Structure that represents the Webhook configuration in the YAML file
+type WebHook struct {
+	Name          string              `mapstructure:"name"`
+	URL           string              `mapstructure:"url"`
+	User          string              `mapstructure:"user"`
+	Password      string              `mapstructure:"password"`
+	CustomHeaders []map[string]string `mapstructure:"custom-headers"`
```

Comment:
Maybe like this?
```suggestion
	CustomHeaders []map[string]string `mapstructure:"custom-headers,omitempty"`
```

Reply:
fixed, but it will have the same result with or without omitempty